### ISSUE TITLE
Google said: API_NOT_AVAILABLE

### ISSUE THREAD

**Issue by brianjmurrell** [1nikolas/play-integrity-checker-app#5]
Hi.

I am trying to run your app and when I click CHECK it reports:

```
Error
Error getting token from Google.  Google
said: API_NOT_AVAILABLE
```

This was at least running before I installed Magisk 25.2.

As an additional data item, Google Play Store is also reporting that I am offline even though I really am.  Many online functions are working perfectly fine.

I have done the typical Magisk Deny List application to hide root from Google services and installed the universal safetynet fix module v2.3.1.

Anything more I need to do?

**Comment by 1nikolas** [comment]
![image](https://user-images.githubusercontent.com/30593419/185087456-e7ad67c2-21df-4dbd-8204-c1cf678c4db0.png)

Old playstore probably

**Comment by 1nikolas** [comment]
Moved to #9
